Sources/AppcuesKit/Presentation/ExperienceLoader.swift

```diff
@@ -17,6 +17,13 @@ internal protocol ExperienceLoading: AnyObject {
         trigger: ExperienceTrigger,
         completion: ((Result<Void, Error>) -> Void)?
     )
+
+    func loadPush(
+        id: String,
+        published: Bool,
+        queryItems: [URLQueryItem],
+        completion: ((Result<Void, Error>) -> Void)?
+    )
```

Comment:
slightly funky looking to have push loading in `ExperienceLoading`. It would also seem odd to have in `PushMonitoring`. Maybe some other small util under the `/Push` folder? But not terribly critical. Certainly a small addition here at this point.

Reply:
`ContentLoading` ? or leaving as-is is fine with me too, just came to mind looking it over.

Reply:
`ContentLoading` is good. I'll sleep on it and use that in the morning if nothing better comes to mind.